client/blocks/nav-unification-modal/style.scss

```
@@ -169,6 +169,8 @@ $modal-footer-height: 30px + ( $modal-footer-padding-v * 2 );
 .nav-unification-modal__picture {
 	display: flex;
 	margin-top: 20px;
+	// Safari default for align-items is stretch
```

```suggestion
	// Preserve image aspect ratio on Safari (default for align-items is stretch)
```

This provides the "why" for future devs.

---

Thanks. I've just updated that, good shout.